packages/terra-section-header/src/SectionHeader.module.scss

```diff
@@ -21,6 +21,7 @@
     padding-bottom: var(--terra-section-header-padding-bottom, 0.5rem);
     padding-top: var(--terra-section-header-padding-top, 0.5rem);
     width: 100%;
+    margin: 0;
```

Comment:
why do we need change in margin for section-header ?? I guess we were trying to style only content of section-header

Reply:
This was added to suppress the default margin added by heading element (which was causing wdio failures)